webui/react/src/components/JupyterLabModal.tsx

```diff
@@ -452,7 +454,11 @@ const JupyterLabForm: React.FC<{
           ))}
         </Select>
       </Form.Item>
-      <Form.Item initialValue={defaults?.template} label="Template" name="template">
+      <Form.Item
+        extra={templatesOn && <Link path={paths.templates()}>Manage Templates</Link>}
```

Comment:
**suggestion:** couple of things to help smooth the workflow, with the idea of allow the user to continue where they left off. I think right now this looks like it will just take them to the templates page. What if it opens a new window/tab so they don't lose their changes in the modal (not a big deal but just a small thing). This might be to big to do in this PR, but to change the `Manage Templates` to a `Create Template` flow instead where it takes the user to actually create a template and come right back to this modal with the newly created on.

Reply:
I like the idea of opening a new tab to configure templates, then come back without losing their changes, but without streaming updates, I don't think newly added templates will automatically show up, unless we poll on this page or migrate templates to a store...